migrator/migrator.go

```diff
@@ -236,6 +326,23 @@ func (p *PouchMigrator) save2Disk(homeDir string, meta *pouch.PouchContainer) er
 	return nil
 }
 
+func (p *PouchMigrator) checkVolumeRefs(ctx context.Context, cid string, mounts []*pouchtypes.MountPoint, volumeRefs map[string]string) (map[string]string, error) {
```

Comment:
This file would update the data in the input parameter `volumeRefs`, does it make sense?

Reply:
The parameter `volumeRefs` is to calculate  references by containers of a volume, so this function will change the data of `volumeRefs`.

If you have more reasonable advice, please let me know, thanks